test/ipcFixtures.hpp

```diff
@@ -283,70 +297,6 @@ TEST_P(umfIpcTest, BasicFlow) {
     EXPECT_EQ(stat.closeCount, stat.openCount);
 }
 
-TEST_P(umfIpcTest, GetPoolByOpenedHandle) {
```

Comment:
@bratpiorka compatibility tests failed because memory regions retrieved from the `umfOpenIPCHandle` function do not belong to the pool anymore (as we agreed earlier). That is why I removed this `TEST_P(umfIpcTest, GetPoolByOpenedHandle)` test.

Could you please suggest how to fix the compatibility tests?

Reply:
Please add a filter to the tests in the compatibility workflow with an appropriate comment.

Reply:
Done

Reply:
I have to filter out the test in the compatibility flow (an older version of the tests, where this test is present, is used). However, in the current version, this test is just removed.

I did not get your point regarding the comment, where do you suggest adding the comment?

Reply:
I see, so it is OK now. If there is no skipped test now, there is no need to add a comment.